src/promptflow/promptflow/_cli/_pf_azure/_run.py

```diff
@@ -353,6 +355,31 @@ def add_parser_run_update(subparsers):
     )
 
 
+def add_parser_run_download(subparsers):
```

Comment:
could you share a screenshot of file structure of a downloaded run (image scenario)?